kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue

```diff
@@ -34,6 +34,7 @@
       <SearchBox
         key="channel-search"
         placeholder="findSomethingToLearn"
+        :tabindex="0"
```

Comment:
What was the reason for adding `tabindex="0"` on the search box `form` element? Wondering if this may be related to problems with two TAB key presses needed.

Reply:
Yes - this is definitely the reason. I originally put this here because I was trying to get the focus _into_ the search field, but I realize now that I could just find that via the input attribute of the form. I'll fix that. Thank you!